docs/AUDIO.md

```diff
@@ -0,0 +1,146 @@
+Ambisonics | Channel layout that encodes fully spherical spatial audio as a set of spherical harmonic basis function coefficients |
+
+
+### - Audio sensor specs
+
+|Config name|Data Type|Default Value|Usage|
+|-----------|---------|-------------|-----|
+|uuid|string|""|uuid to identify the sensor object|
+| outputDirectory | string | "" | Output directory prefix for the simulation. Folders with outputDirectory + i should be created if you want to dump the wave files. (i = 0 indexed simulation iteration |
+| acousticsConfig | [RLRAudioPropagationConfiguration()](#acoustics-configuration) |  Defined in the relevant section | Acoustic configuration struct that defines simulation parameters |
+| channelLayout | [RLRAudioPropagationChannelLayout()](#channel-layout) |  Defined in the relevant section | Channel layout for simulated output audio |
+
+
+
+### - APIs
+
+The audio sensor is build in C++ and exposed on pythom via py-bindings. Import the following to get access to the audio sensor
+
+
+|python imports|
+|------|
+|import habitat_sim|
+|import habitat_sim._ext.habitat_sim_bindings as hsim_bindings|
+|import habitat_sim.sensor|
+|import habitat_sim.sim|
+
+
+The various structs and enums can be accessed as part of the hsim_bindings.
+
+|struct/enum in hsim_bindings|notes|
+|------|-----|
+|hsim_bindings.RLRAudioPropagationConfiguration() | acoustic configs struct |
+|hsim_bindings.RLRAudioPropagationChannelLayout() | channel layout struct |
+|hsim_bindings.RLRAudioPropagationChannelLayoutType.Binaural | channel type enum |
+
+The acoustic sensor spec is part of habitat_sim
+
+|struct/enum in habitat_sim|notes|
+|------|-----|
+|habitat_sim.AudioSensorSpec() | acoustic sensor spec |
+
+
+To call apis on the audio sensor, get access to the audio sensor object using the uuid.
+
+|apis for audio_sensor|notes|
+|------|-----|
+|audio_sensor = sim.get_agent(0)._sensors["audio_sensor"]| get the audio sensor object from the habitat sim object|
+| audio_sensor.setAudioSourceTransform(np.array([x, y, z])) | set the audio source location where x,y,z are floats|
+|audio_sensor.reset() | User can reset the simulation object and start fresh. This is same as deleting the audio sensor.|
+
+Relevant apis on the sim object
+
+|apis for the habitat sim object|notes|
+|------|-----|
+| sim.add_sensor(audio_sensor_spec)| Add the audio sensor. This is similar to adding any other sensors|
+|obs = sim.get_sensor_observations()["audio_sensor"]|Get the impulse response. obs is a n-d array where n = channel count|
+
+
+
+### - Steps to run audio simulation in python
+
+Please see the [audio_agent.py](https://github.com/facebookresearch/habitat-sim/tree/main/examples/tutorials/audio_agent.py) for a sample script on how to use the python audio sensor. Follow the steps and refer to the python script for the code.
```

Comment:
Suggestion, not urgent: maybe this should be an interactive Colab tutorial we link to? Each cell could have the code and tutorial text embedded.
```suggestion
Please see the [audio_agent.py](https://github.com/facebookresearch/habitat-sim/tree/main/examples/tutorials/audio_agent.py) for an example of how to use the python audio sensor. Follow these steps and refer to the python script for the code.
```

Reply:
Ack, ill work with you to get add it to the Colab tutorial